lib/use-checkout-pricing.js

```diff
@@ -132,7 +132,6 @@ export default function useCheckoutPricing(initialInputs = {}, handleError = thr
 
   const output = {
     price: (pricing && cloneDeep(pricing.price)) || {},
```

Comment:
I assumed we still wanted to keep the cloneDeep here but wasn't 100%

Reply:
Yah it needs to be cloned to prevent the consumer from changing pricing values within the underlying `CheckoutPricing`